WORKSPACE

```diff
@@ -26,8 +26,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 # )
 git_repository(
     name = "io_bazel_rules_go",
-    remote = "https://github.com/bazelbuild/rules_go.git",
     commit = "f2373c9fbd09586d8e591dda3c43d66445b2d7ca",
+    remote = "https://github.com/bazelbuild/rules_go.git",
```

Comment:
We *might* be able to remove this hack now that rules_go v0.18.7 has been released

Reply:
Do you want me to do it as part of this PR?

Reply:
No, I can do that as a followup